client/my-sites/plans-features-main/index.tsx

```diff
@@ -722,6 +728,48 @@ const PlansFeaturesMain = ( {
 		[]
 	);
 
+	const handlePlanTypeSelectorChange = useCallback(
+		( selectedItem: { key: SupportedUrlFriendlyTermType } ) => {
+			const additionalPathProps = {
+				...( planTypeSelectorProps.redirectTo
```

Comment:
Similar to the other comment https://github.com/Automattic/wp-calypso/pull/86127/files#r1450171723
Is `redirectTo` used anywhere beyond `plans-features-main` by the `PlanTypeSelector` components?

Reply:
https://github.com/Automattic/wp-calypso/pull/86127#discussion_r1450955200

Reply:
It looks like the `redirectTo` props doesn't even exist in PlansFeaturesMain or 😆 Removing references to it entirely in our new hook usePlanTypeDestinationCallback. I'll clean other references up in the plan type selector component in the follow-up where I refactor the Plan Type Selector Toggle